internal/downloader/download.go

```diff
@@ -111,14 +139,19 @@ func (d *Downloder) download(desc, urlToGet, destination string, mode os.FileMod
 			resp.Status,
 		)
 	}
+	temporaryDestinationFile, err := ioutil.TempFile(os.TempDir(), "kuberlr-kubectl-");
+	if err != nil {
+		return fmt.Errorf("Error trying to create temporary file in %s: %v", os.TempDir(), err)
+	}
 
-	f, err := os.OpenFile(destination, os.O_CREATE|os.O_WRONLY, mode)
+	f, err := os.OpenFile(temporaryDestinationFile.Name(), os.O_CREATE|os.O_WRONLY, mode)
 	if err != nil {
 		return fmt.Errorf(
 			"Error while downloading %s to %s: %v",
-			urlToGet, destination, err)
+			urlToGet, temporaryDestinationFile.Name(), err)
 	}
```

Comment:
These lines are no longer needed: `ioutil.Tempfile` already returns an open `*os.File`, which can be used to write to the file.

Reply:
Done in commit f0a80e7